target/linux/ramips/image/mt76x8.mk

```diff
@@ -352,6 +352,14 @@ define Device/wavlink_wl-wn570ha1
 endef
 TARGET_DEVICES += wavlink_wl-wn570ha1
 
+define Device/wavlink_wl-wn575a3
```

Comment:
Please include a SUPPORTED_DEVICES += wl-wn575a3 so older installations can upgrade without a forced sysupgrade.

Reply:
Oh cool, I didn't realise that was an option. Fixed.